addon/templates/scopes/edit.hbs

```diff
@@ -1,103 +1,119 @@
-          </th>
-          <th>
-            {{t "emeis.acl-table.headings.user"}}
-          </th>
-          <th>
-            {{t "emeis.acl-table.headings.role"}}
-          </th>
-        </table.head>
-        {{#if @model.isNew}}
-          <tbody>
-            <tr>
-              <td colspan="99" class="uk-text-center">{{t "emeis.scopes.aclHint"}}</td>
-            </tr>
-          </tbody>
-        {{else}}
-          <table.body
-            @emptyIcon="warning"
-            @emptyIconColor="warning"
-            @emptyWarning={{t "emeis.scopes.emptyAclList"}}
-            as |body|
-          >
-            <body.row>
-              {{#let body.model as |aclEntry|}}
-                <td data-test-acl-name={{aclEntry.id}}>
-                  {{concat aclEntry.user.firstName " " aclEntry.user.lastName}}
-                </td>
-                <td data-test-acl-username={{aclEntry.id}}>
-                  {{aclEntry.user.username}}
-                </td>
-                <td data-test-acl-role={{aclEntry.id}}>
-                  {{aclEntry.role.name}}
-                </td>
-              {{/let}}
-            </body.row>
-          </table.body>
-        {{/if}}
-      </DataTable>
-    </div>
-</div>
\ No newline at end of file
+    </EditForm>
+  </div>
+  <div class="uk-width-1-2@xl">
+    <DataTable
+      @heading={{t "emeis.scopes.edit.acl.title"}}
+      @modelName="acl"
+      @filter={{this.queryParamsfilter}}
+      @include="user,role,scope"
+      @page={{this.page}}
+      @search={{this.search}}
+      @updatePage={{fn this.updateQueryParam "page"}}
+      @updateSearch={{fn this.updateQueryParam "search"}}
+      as |table|
+    >
+      <table.head>
+        <th>
+          {{t "emeis.acl-table.headings.name"}}
+        </th>
+        <th>
+          {{t "emeis.acl-table.headings.user"}}
+        </th>
+        <th>
+          {{t "emeis.acl-table.headings.role"}}
+        </th>
+      </table.head>
+      {{#if @model.isNew}}
+        <tbody>
+          <tr>
+            <td colspan="99" class="uk-text-center">{{t
+                "emeis.scopes.aclHint"
+              }}</td>
+          </tr>
+        </tbody>
+      {{else}}
+        <table.body
+          @emptyIcon="warning"
+          @emptyIconColor="warning"
+          @emptyWarning={{t "emeis.scopes.emptyAclList"}}
+          as |body|
+        >
+          <body.row>
+            {{#let body.model as |aclEntry|}}
+              <td data-test-acl-name={{aclEntry.id}}>
+                {{concat aclEntry.user.firstName " " aclEntry.user.lastName}}
+              </td>
+              <td data-test-acl-username={{aclEntry.id}}>
+                {{aclEntry.user.username}}
+              </td>
+              <td data-test-acl-role={{aclEntry.id}}>
+                {{aclEntry.role.name}}
+              </td>
+            {{/let}}
+          </body.row>
+        </table.body>
+      {{/if}}
+    </DataTable>
+  </div>
+</div>
```

Comment:
Those big diffs are kind of messing up our git history. Maybe it would be cleaner to

- create a new PR "format templates with prettier"
- check correct formatting in CI
- format all templates

Reply:
Update: I went ahead and did that: https://github.com/projectcaluma/ember-emeis/pull/661